NuGet.Config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This shouldn’t be required. The feed details are inherited in a props file via the shared infrastructure sub module

Reply:
Ohh good to know, removed it